docs/triage-tool.md

```diff
@@ -20,6 +20,10 @@ The tool follows a three-step process:
      failing, and a reference commit of XLA (JAX) that can be used to reproduce the
      regression.
 
+The third step can also be used on its own, via the `--start-container` and
+`--end-container` options, which allows it to be used between private container tags,
+without the dependency on the `ghcr.io/nvidia/jax` registry.
```

Comment:
Maybe specify that we suppose those container are based on our JAX-Toolbox container and  so has XLA under /opt/xla, JAX under /opt/jax and has the build-jax.sh script? If I forgot other expectations, just add them.

Reply:
Done. I don't immediately know of any others.